tests/conftest.py

```diff
@@ -107,6 +122,71 @@ def fmt_with_cell_metadata(request):
     return request.param
 
 
+def pytest_runtest_setup(item):
+    for mark in item.iter_markers():
+        for tool in [
+            "jupytext",
+            "black",
+            "isort",
+            "flake8",
+            "autopep8",
+            "jupyter nbconvert",
+        ]:
+            if mark.name == f"requires_{tool.replace(' ', '_')}":
+                if not tool_version(tool):
+                    pytest.skip(f"{tool} is not installed")
```

Comment:
For another PR, are these necessary? I think it would be clearer to have them inside the tests themselves

Reply:
In the tests I use these tools on the command line, that's why I use `tool_version` here to determine whether they are available. I have seen you comment re `import/skip` below but I am not sure this would cover all the tools here.